prusti-interface/src/specs/typed.rs

```diff
@@ -254,6 +254,8 @@ pub struct TypeSpecification {
     pub source: DefId,
     pub invariant: SpecificationItem<Vec<DefId>>,
     pub trusted: SpecificationItem<bool>,
+    pub has_model: Option<(String, LocalDefId)>,
```

Comment:
`has_model` sounds like a `bool`, not an `Option`.

Reply:
I changed it to just `model`.